### DISCUSSION TITLE
[Feature] Can scoop list display all installed versions, so that nvm can be completely replaced for nodejs version management?

### DISCUSSION THREAD

**Discussion by tianlongJ** [ScoopInstaller/Scoop#5708] [Help and Questions]
Can scoop list display all installed versions, so that nvm can be completely replaced for nodejs version management?

**Comment by HUMORCE** [comment]
`scoop info nodejs` or `(scoop info nodejs).installed`

> nvm can be completely replaced for nodejs version management?

If you just need to switch nodejs versions manually, then this is possible. nvm has more features for nodejs management.

**Reply by tianlongJ** [reply]
This has no effect. scoop info nodejs only displays the latest version, not all

**Reply by HUMORCE** [reply]
```
❯ scoop info nodejs-lts

Name        : nodejs-lts
Description : As an asynchronous event driven JavaScript runtime, Node is designed to build scalable network applications.
              (Long Term Support)
Version     : 20.9.0
Bucket      : main
Website     : https://nodejs.org
License     : MIT
Updated at  : 10/24/2023 4:26:42 PM
Updated by  : github-actions[bot]
Installed   : 18.18.2
              20.9.0
Path Added  : <root>\bin
              <root>


~
❯ (scoop info nodejs-lts).installed
18.18.2
20.9.0
```

**Reply by tianlongJ** [reply]
I don’t have the installed attribute here

![image](https://github.com/ScoopInstaller/Scoop/assets/50390874/e80db3d5-64dc-4244-a99d-f8094fc1c163)

**Reply by HUMORCE** [reply]
scoop which node

**Reply by tianlongJ** [reply]
![image](https://github.com/ScoopInstaller/Scoop/assets/50390874/d355afe3-3201-49bf-88a4-969385f1b150)

**Reply by tianlongJ** [reply]
I found the problem. If I installed it using @version number first, there would be no installed. If I installed it using the package name first, there would be

**Reply by HUMORCE** [reply]
https://github.com/ScoopInstaller/Versions

Installing them with manifests from Versions bucket.

```
scoop install versions/nodejs16
scoop install versions/nodejs18
scoop install main/nodejs-lts

node --version
v18.x
scoop reset nodejs16
node --version
v16.x
scoop reset nodejs18
node --version
v18.x
```

**Comment by tianlongJ** [comment]
It can be turned off. I modified the code myself to achieve it.
